source/dofs/dof_tools.cc

```diff
@@ -1546,12 +1524,8 @@ namespace DoFTools
       }
     else
       {
-        for (typename DoFHandler<dim, spacedim>::active_cell_iterator cell =
-               dof_handler.begin_active();
-             cell != dof_handler.end();
-             cell++)
-          if (cell->is_locally_owned())
-            cell_owners[cell->active_cell_index()] = cell->subdomain_id();
+        for (const auto &cell : dof_handler.active_cell_iterators())
+          cell_owners[cell->active_cell_index()] = cell->subdomain_id();
```

Comment:
you accidentally ate the `if (cell->is_locally_owned())`

Reply:
Good observation, thanks for this and the other comment. Let's try again.